### ISSUE TITLE
Action not work on 3.6.3 version

### ISSUE THREAD

**Issue by chicco27** [mwiedemeyer/ProxySwitcher#15]
I've update from 3.6.1 to 3.63 version and the action button dosen't work. When I push the botton "add action" there aren't Firefox or Internet Explorer buttons.

**Comment by mwiedemeyer** [comment]
Please make sure, that you have the AddIn subfolder and it contains the addin dll's

**Comment by AndrewBuis** [comment]
I had the same issue, posted to a different thread. 
Looks like extracting the zip and running the EXE doesn't register the dlls on first run. Searching regedit, I have:
C:|Program Files (x86)|Proxy Switcher|Addins|ProxySwitcher.Actions.DefaultActions.dll
Then the entry:
ProxySwitcher.Actions.DefaultActions,Version="3.6.1.0",Culture="neutral",PublicKeyToken="C62B3978F9D76BE9",ProcessorArchitecture="MSIL"

Is there a .msi version available?

**Comment by chicco27** [comment]
Yes the files are present on the subfolder but as @AndrewBuis said on EXE run they aren't loaded.

**Comment by Notrede** [comment]
Hi and thanks for the work...

Same here on Win10... no actions in the menu...

**Comment by madlou** [comment]
I have the same issue; i extracted the zip over the previous installation and now I am unable to add an action.

**Comment by madlou** [comment]
Not having done any .net development ever, but wanting this tool to work for me... I got it working for me by:

1. Downloading the source from here
2. Downloading Visual Studio Community 2015 (VSC)
3. Building the project in VSC
4. Copying the folders/files from "\ProxySwitcher-master\ProxySwitcher.Actions.DefaultActions\bin\Debug" to "\ProxySwitcher-master\ProxySwitcher\bin\Debug\Addins"
5. Running "\ProxySwitcher-master\ProxySwitcher\bin\Debug\ProxySwitcher.exe"

Overkill... well, probably.

I appreciate that this may not help anyone else, but thought I'd share.

**Comment by NorbertMK** [comment]
Fixed addins in 3.6.3:
[https://github.com/NorbertMK/ProxySwitcher/releases/download/3.6.3/Release.zip](url)
[Release.zip](https://github.com/mwiedemeyer/ProxySwitcher/files/561785/Release.zip)
In post-build I found:
> xcopy "$(TargetPath)" "..\..\..\ProxySwitcher\bin\Debug\AddIns" /y
> xcopy "$(TargetDir)\de" "..\..\..\ProxySwitcher\bin\Debug\de" /y
In all addins (standard + 6 in actions) I have set this to set:
> xcopy /y /d  "$(TargetPath)" "$(SolutionDir)ProxySwitcher\$(OutDir)\Addins"
I will try to fix the IE "exceptions list into script field" issue if it persists in IE11.
Also Opera seems broken ("%appdata%/opera" is now "opera software").
